guacamole-docker/README.md

```diff
@@ -12,8 +12,12 @@ How to use this image
 =====================
 
 Using this image will require an existing, running Docker container with the
-[guacd image](https://registry.hub.docker.com/u/glyptodon/guacd/), and another
-Docker container providing either a PostgreSQL or MySQL database.
+[guacd image](https://registry.hub.docker.com/u/glyptodon/guacd/), and either
+network access to a working LDAP server, or another Docker container providing
+a PostgreSQL or MySQL database.
+
+SQL Based Authentication
```

Comment:
The main reason I find this odd is that the outline of the document, taking header level into account, is now the following:

1. What is Apache Guacamole?
2. How to use this image
3. SQL Based Authentication
    1. Deploying Guacamole with PostgreSQL authentication
    2. Deploying Guacamole with MySQL authentication
    3. Deploying Aoache Guacamole with LDAP authentication